app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,6 @@
+module Api
+  module V1
+    class BaseController < ApplicationController
```

Comment:
At the end, this is your ApiController, isn't it?

Reply:
Ok, I will move everything in ApiController to that one.